sw/host/opentitanlib/src/otp/lc_state.rs

```diff
@@ -24,6 +24,15 @@ pub struct LcState {
     secded: LcSecded,
 }
 
+#[repr(u32)]
```

Comment:
nit: should we bindgen these as well?

If not, can you at least add a comment here and the corresponding header file?